packages/@aws-cdk/aws-codecommit/README.md

```diff
@@ -37,6 +37,30 @@ To add an Amazon SNS trigger to your repository:
 repo.notify('arn:aws:sns:*:123456789012:my_topic');
 ```
 
+## Initialize with Code
+
+It is possible to initialize the Repository with an Asset.
+For learning how assets work, see the CDK Assets documentation.
+Be aware, that this will only work on *directories*, not on single files.
+This is due to the fact, that CodeCommit expects a zip in the S3 Bucket.
```

Comment:
I think this explanation begs the question: "what if I create an asset from a .zip file?"

Reply:
that would work too